invenio_jobs/services/services.py

```diff
@@ -528,23 +534,26 @@ def search(self, identity, params):
             None,
             permission_action="read",
         )
-        MAX_DOCS = 50_000
-        BATCH_SIZE = 1_000
+        max_docs = current_app.config.get("JOBS_LOGS_MAX_RESULTS", 1_000)
```

Comment:
Having the max and batch sizes with the same values will defeat the batching advantages, no?